### DISCUSSION TITLE
如何调用 错误样式的 MyLoading控件

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#5298] [提问] [answered]
如题
想调用错误样式的Loading
根据源代码添加了IsError
PCL成功炸了.jpg
![0a827d2a6bc2632dc171694e77af5ce1](https://github.com/user-attachments/assets/6e04663f-339c-4360-b5f1-9258a9260aaa)
![d9af91a7643cb3232f47c70ae29d1b53](https://github.com/user-attachments/assets/93dce3db-953f-41de-8a1d-3c7be66580a4)

**Comment by LTCatt** [comment, marked as answer]
没有考虑过……
如果确实要用的话可以自行把它的矢量图复刻一个……
